Gruntfile.coffee

```diff
@@ -1329,6 +1345,15 @@ module.exports = (grunt) ->
 				options:
 					testname: "Local Test - <%= grunt.template.today('yyyy-mm-dd hh:MM') %>"
 
+		"string-replace":
+			inline:
+				files:
+					'dist/wet-boew/js/': 'dist/wet-boew/js/*.js'
+				options:
+					replacements: [
```

Comment:
@nathanv we are scheduled to release WET next Tuesday (August 15th), any chance this PR will be updated/ready by then?